justfile

```diff
@@ -97,7 +97,8 @@ run args="":
       --dart-define="BRANCH=$(git rev-parse --abbrev-ref HEAD)" \
       --dart-define="REGTEST_FAUCET=http://localhost:8080" \
       --dart-define="REGTEST_MAKER_FAUCET=http://localhost:18000/api/pay-invoice" \
-      --dart-define="HEALTH_CHECK_INTERVAL_SECONDS=2"
+      --dart-define="HEALTH_CHECK_INTERVAL_SECONDS=2" \
+      | tee -a ../data/app.log
```

Comment:
This will only redirect stdout and not stderr: maybe could do something like https://stackoverflow.com/a/53051506 ?

Reply:
Also, is always appending what we want?

Reply:
> This will only copy stdout to the logfile and not stderr: maybe could do something like https://stackoverflow.com/a/53051506 ?

indeed, I forgot to redirect stderr to stdout with `2>&1` before the pipe . I'll try it out to see if it works correctly (with our interactive terminal etc.)

Reply:
> Also, is always appending what we want?

I'd say yes, it's a laughably small amount of space compared to our build. I can't imagine this becoming too unwieldy. I can also ensure I wipe it when we do `just wipe` if you think it would be a good idea.

Reply:
> Also, is always appending what we want?

I removed the appending. We typically would be interested in the last run.

Reply:
> This will only copy stdout to the logfile and not stderr: maybe could do something like https://stackoverflow.com/a/53051506 ?

I tried with using stderr as well, but I decided against it. all the logs we're outputting are on stdout, and we get ~800 lines of warnings from Flutter using deprecated code en stderr in the begininng that really make it not the preferred choice 😅 (but if we did it, it would be the way I mentioned above).

Reply:
Ah, I see. No problem then! Are we sure that Flutter's exception logs will be printed to stdout?

Reply:
to be honest, I can't invest more time into it. we'll see when we get there. it's still an improvement IMHO ;D

Reply:
Yep agree!